server/src/modules/auth/auth.controller.ts

```diff
@@ -33,27 +35,18 @@ export class AuthController {
     this.userService = userService
   }
 
-  loadUser = async (req: Request, res: Response): Promise<Response> => {
+  loadUser: ControllerHandler<unknown, UserType | null | Message> = async (
```

Comment:
Why is there the need for null?

Reply:
Sequelize and hence `UserService.loadUser()` could possibly return null